elastic/kibana
# Replace Sass `@euiButton` mixin usage with CSS-in-JS

## Summary

This PR removes a Sass mixin usage that is shortly to be deprecated/removed from EUI.

I think this somewhat addresses #122594 / #122556 but to be honest I'm not 100% sure what's going on with that meta issue 😅 

### QA

| Before | After |
|--------|--------|
| <img width="231" alt="" src="https://github.com/user-attachments/assets/578ea6d8-1dce-4daa-9e3d-e1aac7079ec3"> | <img width="260" alt="" src="https://github.com/user-attachments/assets/482aa5a1-ed0c-4252-95b0-8b324628cb6c"> | 

Note that the height of the button has changed from prod, but from what I can tell this is actually correct and was previously broken on production. Hover/focus styles should remain the same compared to production.

- `yarn storybook shared_ux`
- Go to http://localhost:9001/?path=/story/button-exit-full-screen-button--exit-full-screen-button

### Checklist
